### PR TITLE
fix(console): reset password label

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Fix reset password label.

## Testing

截屏2022-06-29 下午2 44 38

### CODE REVIEW

File: packages/console/src/pages/UserDetails/components/CreateSuccess.tsx [hunk 1]
```diff
@@ -17,9 +17,10 @@ type Props = {
   password: string;
   title: AdminConsoleKey;
   onClose: () => void;
+  passwordLabel?: string;
```
Comment (xiaoyijun) [inline packages/console/src/pages/UserDetails/components/CreateSuccess.tsx:20]: Maybe we could use `AdminConsoleKey` as its type?

Comment (wangsijie) [reply packages/console/src/pages/UserDetails/components/CreateSuccess.tsx:20]: It'll be too complicated for string template in `navigator.clipboard.writeText`

Comment (demonzoo) [reply packages/console/src/pages/UserDetails/components/CreateSuccess.tsx:20]: I think using AdminConsoleKey is fine. If the code is too complicated in  `clipboard.writeText`, we can consider extracting these clauses into several consts first. E.g.

```tsx
const usernameRow = `${t('user_details.created_username')} ${username}`;
const passwordRow = `${t(passwordlabel ?? 'user_details.created_password')} ${password}`;

await navigator.clipboard.writeText(`${usernameRow}\n${passwordRow}`);
```

**Review by demonzoo — APPROVED** [review]
Overall looks good. Please take a look at the comments

File: packages/console/src/pages/UserDetails/components/CreateSuccess.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/UserDetails/components/CreateSuccess.tsx:20]: 截屏2022-07-01 下午12 59 32

Comment (xiaoyijun) [reply packages/console/src/pages/UserDetails/components/CreateSuccess.tsx:20]: Leave a discussion issue.
Just move on.

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

**Pull request merged**